src/Core/Content/Mail/Service/MailFactory.php

```diff
@@ -38,13 +38,8 @@ public function create(
             ->to(...$this->formatMailAddresses($recipients))
             ->setMailAttachmentsConfig($additionalData['attachmentsConfig'] ?? null);
 
-        foreach ($contents as $contentType => $data) {
-            if ($contentType === 'text/html') {
-                $mail->html($data);
-            } else {
-                $mail->text($data);
-            }
-        }
+        $mail->html($contents['text/html']);
```

Comment:
Is this the same? At least the phpstan type on `AbstractMailFactory` only states that it can have both keys, not should. Should we update that to something like: `array{'text/plain': string, 'text/html': string} $contents` ? (as long as they both really are required).

Reply:
For now I adjusted it as mentioned, let me know if it should be further adjusted.

Reply:
it is good like this :+1: